mino/minows/key/mod.go

```diff
@@ -0,0 +1,63 @@
+package key
+
+import (
+	"crypto/rand"
+	"github.com/libp2p/go-libp2p/core/crypto"
+	"go.dedis.ch/dela/core/store/kv"
+	"golang.org/x/xerrors"
+)
+
+// Storage provides persistent Storage for private keys.
+type Storage struct {
```

Comment:
`Storage` alone seems sufficient here, as it's inside `key` package and will be referenced as `key.Storage` when used. seems consistent with go naming convention [here](https://go.dev/doc/effective_go#package-names): "For instance, the buffered reader type in the bufio package is called Reader, not BufReader, because users see it as bufio.Reader".